frontend/src/lib/utils.tsx

```diff
@@ -1276,3 +1276,9 @@ export function isEllipsisActive(e: HTMLElement | null): boolean {
 export function isGroupType(actor: ActorType): actor is GroupActorType {
     return actor.type === 'group'
 }
+
+export function noop(): void {}
```

Comment:
I'm a bit against adding `noop` to utils – IMO such function should be a last resort if there's just no way to handle disabled behavior in a more obvious way, not a useful helper like utils in general

Reply:
sure I can bring it back into the component file